apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Collaborative code generation with LLMs'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we developed a python package pseudocode which much like its name implies allows developers to annotate their code with pseudocode and produce testable code generated by LLMs without the user writing any code.'
+category: ["ML", "LLM", "python", "openai"]
+featuredImage:
+  src: '/posts/openai-pseudocode/pseudocode-screenshot.png'
+  alt: 'A screenshot of the pseudocode program running'
+hero:
+  imageSrc: /posts/hero-paris.webp
+  imageAlt: 'Data visualization of Paris city'
+---
+
+<base target="_blank" />
+
+Large Language Models (LLMs) have revolutionized the field of natural
+language processing (NLP) and artificial intelligence (AI), opening up
+a world of possibilities across various domains. These models have
+been trained on vast amounts of data and are capable of generating
+human-like responses to a wide range of queries. The developer
+community has seen especially good results due to the large amount of
+source code available online for training.
+
+Github's Co-pilot is a tool which will autocomplete code for a
+developer but lacks the ability to interact with the completation,
+provide feedback, and enforce a test suite on the generated code.
+
+As an experiment we developed a python package
+[pseudocode](https://github.com/Quansight/pseudocode) which much like
+its name implies allows developers to annotate their code with
+pseudocode and produce testable code generated by LLMs without the
+user writing any code. We hope that
+[pseudocode](https://github.com/Quansight/pseudocode) may be a "higher
+level language" for writing python code. Below you will find an
+example. We emphasize interfaces via type annotations and function
+docstrings and provide easy ways to have automated tests.
+
+```python
+from pseudocode import pseudo_function
```

Comment:
not a great example. 

it would be better to do what you did your live demo. walk through the process of downloading the issues from the gh api and doing a q&a with it and getting the final result. the sum_func is very underwhelming.